css/main.css

```diff
@@ -0,0 +1,248 @@
+  max-width: 80vw; /*sets a max view width on viewer's screen similarly to percent, responsive*/
+  padding: 10px; /*padding around body element*/
+  background-color: black;
+}
+
+/* a container holding the page elements*/
+#containerblue {
+  display: flex;
+  flex-direction: column; /*sets the flex direction to column - it would be row by default. widths and heights will apply to the column as the main axis is horizontal for column flex direction*/
+  justify-content: stretch;
+  align-items: center;
+  flex: 1; /*allows container tag to take up entire container width*/
+  margin: 0; /*no margin around the container*/
+  max-width: 80vw; /*sets height and width to specified view width*/
+  height: 40vw;
+  background-color: navy;
+  padding: 5px;
+}
+
+/*Gives the h1 content font properties and some padding*/
+h1 {
+  font-size: 30px;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  color: rgb(212, 42, 228);
+  padding-bottom: 20px;
+}
+
+/*css to set  wrap image in a background color of aqua and box shadow and set animation timing*/
+#homeimg {
+  text-align: center;
+  width: 30vw;
+  margin-bottom: 15px;
+  background-color: aqua;
+  box-shadow: 0.5vw 1vh; /*controlling view width and height of box shadow*/
+  padding: 3vh 5vw 2vh 5vw; /*top, left, bottom, right padding*/
+  transition: all 0.5s ease; /*what is being targeted, time, animation timing-accelerates*/
+  transition-property: width, background-color; /*properties that will transition with the action (pseudoclass/id)*/
+}
+/*--home page image hover animation that causes image to grow and background color to change to pink when user hovers over it then it returns to normal when they are no longer hovering--*/
+#homeimg:hover {
+  width: 40vw; /*original width was 35, this makes it pop out 5vw with hover*/
+  background-color: rgb(
+    243,
+    49,
+    178
+  ); /*color that background will change to with hover*/
+  transition-property: width, background-color; /*properties that will transition with the action (pseudoclass/id)*/
+  transition: all 0.5s ease; /*what is being targeted, time, animation timing-accelerates*/
+}
+/*--------Button and Audio styling css--------*/
+/*places audio content in a flex container oriented as a column with the content centered and max of 20 view widths with a small bottom padding to give space for the button*/
+#audio {
+  display: flex;
+  flex-direction: column;
+  justify-content: center; /*centers the content in the flex container*/
+  max-width: 20vw;
+  padding-bottom: 15px;
+}
+
+/*id that targets properties of the button*/
+#button {
+  height: 30px;
+  padding: 4px;
+  border-radius: 10px; /*rounds button edges*/
+  background-color: rgb(243, 49, 178);
+  font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
+  font-weight: bold;
+  color: black;
+}
+
+/*button hover is not working! I want to get help to fix this*/
+button:hover {
+  background-color: purple;
+}
```

Comment:
We can try to work this out sometime. Try adding a "#" before button for now... don't know if that will work or not